tf-modules/archive/variables.tf

```diff
@@ -382,7 +382,7 @@ variable "add_report_topic_policy" {
   default = false
 }
 
-variable "metrics_account_id" {
+variable "report_sns_topic_subscriber_arns" {
   type = list
-  default = []
+  default = ["*"]
```

Comment:
This will allow anonymous user access

Reply:
I think you can use dynamic statement, like in
https://github.com/nasa/cumulus/blob/master/tf-modules/cumulus_distribution/iam.tf#L70
so this way, the policy is always created, and the statement for subscriber is dynamic